cpu/cortexm_common/include/cpu.h

```diff
@@ -58,16 +58,80 @@ extern "C" {
  */
 #define PROVIDES_PM_SET_LOWEST
 
+/**
+ * @name   Pattern to write into the co-processor Access Control Register to
+ *         allow full FPU access
+ *
+ * Used in the `cortexm_init_fpu` inline function below.
+ */
+#define FULL_FPU_ACCESS         (0x00f00000)
```

Comment:
Since this macro is now public, it needs to be renamed to something related to the CPU or file, e.g. CORTEXM_FPU_ACCESS_FULL_MASK